cni/telemetry/client/telemetry_client.go

```diff
@@ -0,0 +1,118 @@
+package telemetryclient
+
+import (
+	"fmt"
+	"os"
+	"sync"
+
+	"github.com/Azure/azure-container-networking/telemetry"
+	"go.uber.org/zap"
+)
+
+const (
+	telemetryNumberRetries          = 5
+	telemetryWaitTimeInMilliseconds = 200
+)
+
+type TelemetryClient struct {
```

Comment:
There's a lot of stutter in the naming. Can't there just be a `telemetry` package with a `Client` type?

Reply:
moved the telemetry client to the existing telemetry package